wwwroot/index.html

```diff
@@ -1,198 +1,385 @@
+                                <g id="Rectangle_10-3" data-name="Rectangle 10" transform="translate(976 1920)" fill="none" stroke="#77C600" stroke-width="2">
+                                    <rect width="260" height="66" rx="33" stroke="none" />
+                                    <rect x="0.5" y="0.5" width="259" height="65" rx="32.5" fill="none" /> </g>
+                                <g id="Rectangle_10-4" data-name="Rectangle 10" transform="translate(1286 1874)" fill="none" stroke="#77C600" stroke-width="2">
+                                    <rect width="260" height="66" rx="33" stroke="none" />
+                                    <rect x="0.5" y="0.5" width="259" height="65" rx="32.5" fill="none" /> </g>
+                                <g id="Rectangle_11" data-name="Rectangle 11" transform="translate(1286 1966)" fill="none" stroke="#77C600" stroke-width="2">
+                                    <rect width="260" height="66" rx="33" stroke="none" />
+                                    <rect x="0.5" y="0.5" width="259" height="65" rx="32.5" fill="none" /> </g>
+                                <path id="Path_7" data-name="Path 7" d="M-279.661,0h1214" transform="translate(321.661 1953)" fill="none" stroke="#77C600" stroke-width="2" />
+                                <path id="Path_6" data-name="Path 6" d="M3502,1902.8h-30v91.527h30" transform="translate(-2216 4.204)" fill="none" stroke="#77C600" stroke-width="2" /> </g>
+                        </svg>
                     </div>
-                    <div class="col-md-6 offset-md-1 position-relative">
-                        <img src="img/screenshots/webpage_ui_Preview_Transaction.png" class="pull-img pull-img-to-right" alt="Wasabi Wallet Transaction Preview" />
+                </div>
+                <div class="row">
+                    <div class="col-lg-3 offset-lg-4"> Blog post: <strong class="d-block mt-2">BITCOIN PRIVACY PRIMER</strong> <a href="https://blog.wasabiwallet.io/bitcoin-privacy-primer/" target="_blank" class="btn btn-secondary affix-arrowright mt-4">Read it here</a> </div>
+                </div>
+                <div class="row">
+                    <div class="col-lg-6 offset-lg-6 text-end">
+                        <h3 class="title-big mt-5 mt-lg-0">reclaim your privacy</h3> </div>
+                </div>
+            </div>
+        </section>
+        <section>
+            <div class="container">
+                <div class="row">
+                    <div class="col-lg-3 offset-lg-1">
+                        <h2 class="title">Privacy by default</h2>
+                    </div>
+                    <div class="col-lg-4">
+                        <p> Get started with good privacy by default. All traffic is routed via Tor. Your wallet transactions are downloaded with block filters. Easily reclaim your privacy with collaborative bitcoin transactions. </p>
```

Comment:
```suggestion
                        <p> Get started with privacy by default. All traffic is routed via Tor by default. Your wallet is synchronized using client-side block filters. Gain on-chain privacy with the built-in coinjoins. </p>
```
this is a bit weird structure imo. my proposal is more of a fix, maybe rewrite is better